core/src/main/res/values/strings.xml

```diff
@@ -428,6 +428,8 @@
     <string name="pref_gpodnet_notifications_sum">This setting does not apply to authentication errors.</string>
     <string name="pref_playback_speed_title">Playback Speeds</string>
     <string name="pref_playback_speed_sum">Customize the speeds available for variable speed audio playback</string>
+    <string name="pref_playback_time_respect_speed_title">Time respect playback speed</string>
```

Comment:
In my opinion, this should either be *Time respect**s*** or *Time**s** respect*. I am not a native English speaker, so feel free to disagree.

Reply:
Since (if I understood correctly) this change only affects the player, can we add that (to avoid confusion as to why the queue doesn't change): `Player time respects playback speed`
Or would that already be too long?

Reply:
I think that's too long. Especially when translating to German :D